scripts/DockerfileNode

```diff
@@ -0,0 +1,72 @@
+COPY packages/neo-one-client-switch/src/ packages/neo-one-client-switch/src
+COPY packages/neo-one-client-switch/package.json packages/neo-one-client-switch/
+COPY packages/neo-one-node-bin/src/ packages/neo-one-node-bin/src
+COPY packages/neo-one-node-bin/package.json packages/neo-one-node-bin/
+COPY packages/neo-one-node-blockchain/src/ packages/neo-one-node-blockchain/src
+COPY packages/neo-one-node-blockchain/package.json packages/neo-one-node-blockchain/
+COPY packages/neo-one-node-browser-worker/src/ packages/neo-one-node-browser-worker/src
+COPY packages/neo-one-node-browser-worker/package.json packages/neo-one-node-browser-worker/
+COPY packages/neo-one-node-browser/src/ packages/neo-one-node-browser/src
+COPY packages/neo-one-node-browser/package.json packages/neo-one-node-browser/
+COPY packages/neo-one-node-consensus/src/ packages/neo-one-node-consensus/src
+COPY packages/neo-one-node-consensus/package.json packages/neo-one-node-consensus/
+COPY packages/neo-one-node-core/src/ packages/neo-one-node-core/src
+COPY packages/neo-one-node-core/package.json packages/neo-one-node-core/
+COPY packages/neo-one-node-data-backup/src/ packages/neo-one-node-data-backup/src
+COPY packages/neo-one-node-data-backup/package.json packages/neo-one-node-data-backup/
+COPY packages/neo-one-node-http-rpc/src/ packages/neo-one-node-http-rpc/src
+COPY packages/neo-one-node-http-rpc/package.json packages/neo-one-node-http-rpc/
+COPY packages/neo-one-node-neo-settings/src/ packages/neo-one-node-neo-settings/src
+COPY packages/neo-one-node-neo-settings/package.json packages/neo-one-node-neo-settings/
+COPY packages/neo-one-node-network/src/ packages/neo-one-node-network/src
+COPY packages/neo-one-node-network/package.json packages/neo-one-node-network/
+COPY packages/neo-one-node-offline/src/ packages/neo-one-node-offline/src
+COPY packages/neo-one-node-offline/package.json packages/neo-one-node-offline/
+COPY packages/neo-one-node-protocol/src/ packages/neo-one-node-protocol/src
+COPY packages/neo-one-node-protocol/package.json packages/neo-one-node-protocol/
+COPY packages/neo-one-node-rpc-handler/src/ packages/neo-one-node-rpc-handler/src
+COPY packages/neo-one-node-rpc-handler/package.json packages/neo-one-node-rpc-handler/
+COPY packages/neo-one-node-storage-common/src/ packages/neo-one-node-storage-common/src
+COPY packages/neo-one-node-storage-common/package.json packages/neo-one-node-storage-common/
+COPY packages/neo-one-node-storage-levelup/src/ packages/neo-one-node-storage-levelup/src
+COPY packages/neo-one-node-storage-levelup/package.json packages/neo-one-node-storage-levelup/
+COPY packages/neo-one-node-vm/src/ packages/neo-one-node-vm/src
+COPY packages/neo-one-node-vm/package.json packages/neo-one-node-vm/
+COPY packages/neo-one-node/src/ packages/neo-one-node/src
+COPY packages/neo-one-node/package.json packages/neo-one-node/
+
+RUN sudo yarn install --non-interactive
```

Comment:
`sudo` is really necessary?

Reply:
if I don't sudo the `yarn install` I get:
```warning "react-static > eslint-config-react-tools > eslint > table > ajv-keywords@2.1.1" has incorrect peer dependency "ajv@^5.0.0".
error An unexpected error occurred: "EACCES: permission denied, mkdir '/tmp/neo-one/packages/neo-one-node-bin/node_modules'".```

I don't need to sudo the yarn build I can make that change.